be/src/olap/olap_server.cpp

```diff
@@ -205,29 +205,32 @@ void StorageEngine::_disk_stat_monitor_thread_callback() {
 }
 
 void StorageEngine::_check_cumulative_compaction_config() {
-    std::string cumulative_compaction_type = config::cumulative_compaction_policy;
+    std::string cumulative_compaction_type = "";
```

Comment:
Most of these configs are mutable, check them firstly here may be useless. A better way is to check them when any of them are mutated?

Reply:
> Most of these configs are mutable, check them firstly here may be useless. A better way is to check them when any of them are mutated?

OK.